src/components/WeeklyReport/Report.tsx

```diff
@@ -1,213 +1,307 @@
-    const [startDatePickerOpen, setStartDatePickerOpen] = useState<boolean>(false);
-    const [endDatePickerOpen, setEndDatePickerOpen] = useState<boolean>(false);
+  const [startHighlightDates, setStartHighlightDates] = useState<Date[]>(
+    props.defaultStartDate ? DateUtilities.getWeek(props.defaultStartDate) : []
+  );
+  const [endHighlightDates, setEndHighlightDates] = useState<Date[]>(
+    props.defaultEndDate ? DateUtilities.getWeek(props.defaultEndDate) : []
+  );
+  const [startDatePickerOpen, setStartDatePickerOpen] =
+    useState<boolean>(false);
+  const [endDatePickerOpen, setEndDatePickerOpen] = useState<boolean>(false);
 
-    const { orgs } = useContext(OrgsContext);
+  const { orgs } = useContext(OrgsContext);
 
-    const updateReportForm = (field: string, value: any) => {
-        setReportForm({ ...reportForm, [field]: value });
-    }
+  const updateReportForm = (field: string, value: any) => {
+    setReportForm({ ...reportForm, [field]: value });
+  };
 
-    const updateStartDate = (date?: Date | null) => {
-        updateReportForm('startDate', date ? DateUtilities.getStartOfWeek(date) : null);
-        setStartHighlightDates(date ? DateUtilities.getWeek(date) : []);
-    }
+  const updateStartDate = (date?: Date | null) => {
+    updateReportForm(
+      "startDate",
+      date ? DateUtilities.getStartOfWeek(date) : null
+    );
+    setStartHighlightDates(date ? DateUtilities.getWeek(date) : []);
+  };
 
-    const updateEndDate = (date?: Date | null) => {
-        updateReportForm('endDate', date ? DateUtilities.getEndOfWeek(date) : null);
-        setEndHighlightDates(date ? DateUtilities.getWeek(date) : []);
-    }
+  const updateEndDate = (date?: Date | null) => {
+    updateReportForm("endDate", date ? DateUtilities.getEndOfWeek(date) : null);
+    setEndHighlightDates(date ? DateUtilities.getWeek(date) : []);
+  };
 
-    const getOpr = async () => {
-        if (props.defaultOpr) {
-            let person = (await spWebContext.ensureUser(props.defaultOpr)).data;
-            updateReportForm("opr", {
-                text: person.Title,
-                imageInitials: person.Title.substr(person.Title.indexOf(' ') + 1, 1) + person.Title.substr(0, 1),
-                Email: person.Email,
-                SPUserId: person.Id.toString()
-            });
-        }
+  const getOpr = async () => {
+    if (props.defaultOpr) {
+      let person = (await spWebContext.ensureUser(props.defaultOpr)).data;
+      updateReportForm("opr", {
+        text: person.Title,
+        imageInitials:
+          person.Title.substr(person.Title.indexOf(" ") + 1, 1) +
+          person.Title.substr(0, 1),
+        Email: person.Email,
+        SPUserId: person.Id.toString(),
+      });
     }
+  };
 
-    useEffect(() => {
-        getOpr(); // eslint-disable-next-line
-    }, [props.defaultOpr]);
+  useEffect(() => {
+    getOpr(); // eslint-disable-next-line
```

Comment:
I feel like this should be re-written to actually leverage the forwardRef params.  Something like:
let startDatePickerRef = useRef<HTMLInputElement>(null);
let StartDatePickerCustomInput = forwardRef<HTMLInputElement>(
        (_props: {}, ref) => (
      <>
        <Form.Label>Report Start Week</Form.Label>
        <Form.Control
          ref={ref}
          .......


Then pass in the ref <StartDatePickerCustomInput ref={startDatePickerRef} />

Thoughts?